credentialz/credentialz.proto

```diff
@@ -697,3 +700,10 @@ message ServerKeysResponse {
 
 message FinalizeRequest {
 }
+
+message GetPublicKeyRequest {
+}
+
+message GetPublicKeyResponse {
+  PublicKey public_key = 1;
```

Comment:
This should be a `repeated` since more than one key may be present. I.e. SSH does one key per algorithm type so this may return 2 values if you have RSA and ed25519 present.

Reply:
I would vote for adding `KeyType` to both the `GetPublicKeyRequest` and `GetPublicKeyResponse` instead.